checkbox-ng/checkbox_ng/launcher/stages.py

```diff
@@ -212,9 +229,33 @@ def _interaction_callback(self, job, result_builder,
             if cmd == 'set-pass':
                 result_builder.outcome = IJobResult.OUTCOME_PASS
             elif cmd == 'set-fail':
-                result_builder.outcome = IJobResult.OUTCOME_FAIL
+                if (
+                    job_state.effective_certification_status == "blocker"
+                    and not isinstance(result_builder.comments, str)
```

Comment:
This is mad. And i'm not talking about this code but the necessity to have it. I'm guessing this is because of the pod madness...